apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-docs.component.html

```diff
@@ -0,0 +1,86 @@
+<fd-docs-section-title [id]="'default'" [componentName]="'infolabel'">
+	Default Info Label
+</fd-docs-section-title>
+<component-example>
+	<fdp-platform-object-status-example></fdp-platform-object-status-example>
+</component-example>
+<code-example [exampleFiles]="platformDefaultObjectStatusHtmlType"></code-example>
+
+<fd-docs-section-title [id]="'Text'" [componentName]="'objectstatus'">
```

Comment:
generally we use camel casing for id and component name.. it can be 'text' and 'objectStatus'. Same for other places in this file